.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -25,5 +25,9 @@ If applicable, add screenshots and/or console outputs to help explain your probl
 -   OS version:
 -   HTCondor version:
 
+**Priority:** Priority level of this bug [critical, high, medium, low]
+
+**Stakeholders:** Concerned stakeholder(s) of this bug [CMS, FactoryOps, Fermilab, FIFE, IGWN, OSG]
+
```

Comment:
I would add these to the list in the previous section (that I'd rename "Info", I don't remember why I choose "Desktop"). `**Name**` are titles, the list is better for structured info that comes on the same line:
```
-  Priority: ....
```
There could be also a `-  Component:` item (frontend, factory, glidein, documentation, ...) that could help assign the issue.

Reply:
Updated as per the comment above